src/modules/map/components/layerManager/layerManager.css

```diff
@@ -25,6 +25,7 @@ li {
 }
 
 .placeholder {
+    padding: 0.2em;
    /* border: 1px solid var(--header-background-color) */
```

Comment:
do we need line 29 in the future?

Reply:
not really....i interpreted this line as a alternative style from @alsturm to 
`  .divider {
    border-bottom: 1px solid var(--header-background-color);
 }` in layerItem.css